src/utils/synchronization.ts

```diff
@@ -104,15 +104,14 @@ export async function postMetadata(
 ): Promise<MetadataImportResponse> {
     try {
         const params: MetadataImportParams = {
-            importMode: "COMMIT",
+            importMode: additionalParams?.importMode ?? "COMMIT",
```

Comment:
There's no need for this setting here, as in line 113 we're already destructuring ``additionalParams``. So ``importMode: "COMMIT"`` here is the default behvaiour and if ``importMode`` is set it will overwrite the default value in the destructuring.

Reply:
Done!, 
I didn't know that the order of destructuring and assigning fields affected to be default or assigned value, thanks.